crates/cheatcodes/spec/src/vm.rs

```diff
@@ -374,7 +374,7 @@ interface Vm {
     /// Not available on EVM versions before Paris. Use `difficulty` instead.
     /// If used on unsupported EVM versions it will revert.
     #[cheatcode(group = Evm, safety = Unsafe)]
-    function prevrandao(bytes32 newPrevrandao) external;
+    function prevrandao(uint256 newPrevrandao) external;
```

Comment:
this replaces the function, we should overload it instead so this becomes non breaking